scipy/optimize/tests/test_linprog.py

```diff
@@ -2073,17 +2077,20 @@ class TestLinprogHiGHSSimplexDual(LinprogHiGHSTests):
     options = {}
 
     def test_lad_regression(self):
-        '''The scaled model should be optimal but unscaled model infeasible.'''
```

Comment:
I see [here](https://github.com/scipy/scipy/pull/15217/files#r769234451) you mention why this is changing.
Can you add a comment pointing to where this test was added and the upstream fix that resolved it?

Reply:
Here is a note that the scaled/unscaled checks are happening: https://github.com/ERGO-Code/HiGHS/issues/494#issuecomment-990748230

Reply:
Add scipy issue reference